IS/publiccode.yml

```diff
@@ -0,0 +1,109 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+applicationSuite: WSO2
+categories:
+  - it-security
+  - it-service-management
+  - identity-management
+description:
```

Comment:
Adding the awards might be nice. As IS has some recognitions 
 https://docs.italia.it/italia/developers-italia/publiccodeyml-en/en/master/schema.core.html

search for "description/[lang]/awards"

Reply:
Updated accordingly